idd/Energy+.idd.in

```diff
@@ -86799,8 +86798,7 @@ ZoneHVAC:RefrigerationChillerSet,
 
 DemandManagerAssignmentList,
        \extensible:2 Duplicate the last two fields DemandManager Object Type and DemandManager Name
-       \memo a list of meters that can be reported are available after a run on
-       \memo the meter dictionary file (.mdd) if the Output:VariableDictionary has been requested.
+       \memo Demand Manager Assignment List
```

Comment:
How is this a more helpful memo?  Same with the next one.

Reply:
Probably need a better sentence to explain what this object is--I will dig out a little bit more from the IORef descriptions to come up with a proper sentence.

Reply:
Yes, better, thanks.